### DISCUSSION TITLE
White screen after upgrade

### DISCUSSION THREAD

**Discussion by journeyguy** [YOURLS/YOURLS#2969] [General]
Hi, 

Our church just upgraded to 1.8 from below, and now we can’t edit or add new URLs. We get a white screen. All the previous short URLs work.

Any help available?

**Comment by journeyguy** [comment]
Also, you can't click on the "Shorten the URL" button to add a new URL.

**Comment by ozh** [comment]
As replied by email, https://github.com/YOURLS/YOURLS/wiki/Troubleshooting-first-steps
Without more details than "we get a white screen" no one is going to be able to help. Vague problems = vague answers.

**Reply by journeyguy** [reply]
Noted. Here's the errors we're seeing:

1. We can't click on the "Shorten the URL" button. Nothing happens when you click on it.
2. We CAN delete an existing Short URL.
3. We cannot create a new one (see #1).
4. **When you try to edit an existing short URL, it takes you to a white screen.**
5. When you click on the share icon, nothing happens.

**Reply by ozh** [reply]
So, you just don't follow any advice from the wiki page. No client side inspection using browser dev tools ("nothing happens when you click on it"), no server logs ("it takes you to a white screen"), etc... Repeating the problem superficial description without giving insights is useless, dude.

**Comment by journeyguy** [comment]
FWIW... I downgraded to 1.7.9, and it's working again.
